recipe/meta.yaml

```diff
@@ -37,8 +37,12 @@ outputs:
         - libuv
         - git >={{ git }}
       host:
-        - cudatoolkit {{ cudatoolkit }}        #[build_type == 'cuda']
+{% if build_type == 'cuda' %}
+        - cudatoolkit {{ cudatoolkit }}
+        - openmpi {{ openmpi }} cuda{{ cudatoolkit }}*   #[mpi_type == 'openmpi']
+{% else %}
         - openmpi {{ openmpi }}                #[mpi_type == 'openmpi']
+{% endif %}
```

Comment:
I think this could still allow for the cpu version of lightgbm to try and use the cuda version.

Reply:
yes, it does, but we don't use `cpu` in the openmpi cpu build string at the moment. Unless we change that I'm not sure there is a good way to say `not` cuda.

Reply:
Since our goal is to make it more explicit, we could add `cpu` to the openmpi build string.

Reply:
yeah - I think that would be a good idea. Then we can revisit this (and perhaps other feedstocks) with an update to make it explicit.